crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -36,6 +41,7 @@ pub struct Shader {
     // we must store strong handles to our dependencies to stop them
     // from being immediately dropped if we are the only user.
     pub file_dependencies: Vec<Handle<Shader>>,
+    pub validate_shader: ValidateShaders,
```

Comment:
Variable name is singular, type name is plural. They should both be singular: this setting is per-shader, as it should be.